index.html

```diff
@@ -86,127 +93,282 @@ <h1 class="intro">I’m Cobby. Glad to see you!</h1>
         </div>
       </section>
     </div>
+    <!-- desktop -->
     <section>
-      <div class="my-individual-project">
+      <div class="my-individual-project desktop">
         <div class="content-wrapper">
-          <div class="project-snaphot">
-            <img
-              src="./assets/Snapshoot Portfolio.png"
-              alt="porfolio-snapshot"
-            />
+          <div class="project-snapshot even">
+            <img src="./assets/Snapshoot Portfolio 3.png" alt="porfolio-snapshot" />
```

Comment:
- [ ] **[OPTIONAL]** Great job on incorporating the suggestions from the previous reviewer! I have **a small optional change that I would like you to consider adding to your card.** Could you please add **a bit more `padding` to your image?** This will help it align more closely with the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122).  🙋‍♂️

   ### Your design:
   ![image](https://github.com/CobbyElsonfx/Portfolio-Website/assets/107952724/46a50c23-5511-4e25-b7fc-85e810ccabbd)

   ### Figma design:
   ![image](https://github.com/CobbyElsonfx/Portfolio-Website/assets/107952724/8dfdf65f-dd29-4613-9555-6315f31995d5)